scripts/.coveragerc

```diff
@@ -0,0 +1,8 @@
+[run]
+omit =
+    tests
+
+[report]
+exclude_lines =
+    pragma: no cover
+    if __name__ == .__main__.:
```

Comment:
EOF

Reply:
@nandojve @scottwcpg @rodrigopex @dbaluta 
Have cleanup all files relating to current comments from contributors.
Have added our PIC32CXSG 'fixup' files to hal_microchip/pic32c
Have added our .svd files to assist with debugging